src/definitions/event-datamodel.json

```diff
@@ -91,7 +91,7 @@
         "type": "boolean",
         "required": true
       },
-      "token": {
+      "participantId": {
```

Comment:
As it is on the entity participant we can call it id here. participantId is fine for the rest of the system

Reply:
Does this block the PR? I understand that it is kind of redundant calling the id of the participant participantId, but it should be better readable within the db.